### PR TITLE
feat: failover http transport #LSC-58

### PR DESCRIPTION
**Pull request by oopcode**

### CODE REVIEW

**Review by swelf19 — COMMENTED** [review]
need to rename `LCD_ENDPOINT` env variable in docker-compose file

File: internal/client/client.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import (
+	"fmt"
+
+	"github.com/go-openapi/runtime"
+	openapiTransport "github.com/go-openapi/runtime/client"
+	"github.com/lidofinance/terra-monitors/collector/config"
+	"github.com/sirupsen/logrus"
+
+	"github.com/lidofinance/terra-monitors/openapi/client"
+	openapiClient "github.com/lidofinance/terra-monitors/openapi/client"
```
[the remark this answers is not in the document]
Comment (oopcode) [reply internal/client/client.go:12]: fixed

File: docker/env/.lido_terra.testnet.env [hunk 2]
```diff
@@ -1,4 +1,4 @@
-LCD_ENDPOINT=tequila-fcd.terra.dev
+LCD_ENDPOINTS=[fcd.terra.dev]
```
[the remark this answers is not in the document]
Comment (oopcode) [reply docker/env/.lido_terra.testnet.env:1]: fixed

File: collector/monitors/validators_repository_v2_test.go [hunk 3]
```diff
@@ -63,7 +65,7 @@ func (suite *ValidatorsCommissionTestSuite) TestFailedV2ValidatorsRepository() {
 	cfg := NewTestCollectorConfig(testServer.URL)
 	cfg.BassetContractsVersion = config.V2Contracts
 
-	valRepository := NewValidatorsRepository(cfg)
+	valRepository := NewValidatorsRepository(cfg, logging.NewDefaultLogger())
```
[the remark this answers is not in the document]
Comment (oopcode) [reply collector/monitors/validators_repository_v2_test.go:68]: fixed

File: README.md [hunk 4]
```diff
@@ -43,6 +43,12 @@ BASSET_CONTRACTS_VERSION=1
 SERVICE_NAME=lido_terra_mainnet
 ```
 
+**N.B.: you can specify failover endpoints (sorted by priority, max to min) for the `LCD_ENDPOINT` config:**
+
+```
+LCD_ENDPOINT=fcd.terra.dev,scp.terra.dev
```
[the remark this answers is not in the document]
Comment (oopcode) [reply README.md:49]: fixed

**Review by swelf19 — APPROVED** [review]

**Pull request merged**